app/api/events.py

```diff
@@ -394,6 +405,7 @@ def before_update_object(self, event, data, view_kwargs):
         :return:
         """
         if data.get('original_image_url') and data['original_image_url'] != event.original_image_url:
+            validate_original_image_url(data['original_image_url'])
```

Comment:
I think this method will create 2 requests which is not a great way to handle image loads as images can be quite large.

Reply:
Yes, I agree. 
@bhaveshAn It will make two network calls for the same image:
1. To verify the image.
2. In the `create_save_image_sizes` method.

I think what @Kreijstal suggested is sub optimal. Here is my suggestion(same as before):
1. In the `create_save_image_sizes` method just raise the HTTPError/URLError
2. Add a try except block for this method call
``` Python
uploaded_images = create_save_image_sizes(data['original_image_url'], 'event', event.id)
```

Reply:
What would the except method do other than changing the error message, why not let the original error bubble up?

Reply:
I only have one issue. The `create_save_image_sizes` is a helper method that can be used from else where in the project as well(May be in the future). It's against the best practices if any helper method expects itself to be called from a specific part of the code. If we add 
``` Python
raise UnprocessableEntity(
            {'source': 'attributes/original-image-url'}, 'Invalid Image URL'
```
in the helper method, then it suggests that we are expecting the data to be called from the event only. How can a helper method be sure that the attribute that has been passed is coming from ```attributes/original-image-url```. Ideally it shouldn't assume anything. 

@iamareebjamal What do you think ?

Reply:
Is this what you want finally

```
raise UnprocessableEntity(
            {'source': ''}, 'Invalid Image URL'
```

Reply:
I am suggesting what I had suggested you before. You had even updated it in this commit: https://github.com/fossasia/open-event-server/pull/4872/commits/5cd819f8d712a9df11d2284b0a442a171221f0b1